qwik-graphql-tailwind/src/components/file-viewer/index.tsx

```diff
@@ -47,7 +46,7 @@ export const FileViewer = component$(() => {
     return <div>Loading...</div>;
   }
 
-  const extension = path?.split('.').pop();
+  const extension = path?.split('.').pop()?.split('/').shift();
```

Comment:
I am trying to understand why an additional `split()` and `shif()` was added.
Is there a scenario where `.split('.').pop()` doesn't work?

Reply:
is kind complicated but if the url has `.yml/` need only the `.yml` not the `/`, that's why.

Reply:
this ticket also handle this issues https://github.com/thisdot/starter.dev-github-showcases/issues/1085